### PR TITLE
fix: dialogflow integration in docker environments

### PR DESCRIPTION
**Pull request by vishnu-narayanan**
# Pull Request Template

## Description

Default installation of Dialogflow installs `protobuf` and `grpc` gems which are compiled against `glibc`. Chatwoot docker image is based on alpine and thus `musl` was having conflicts causing the Dialogflow integration to be broken.

This PR forces `protobuf` and `grpc` to be compiled natively to play nice with alpine.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Tested manually logging into rails console from resulting docker image
- Tested on a docker based env with full end to end test with Dialogflow integration

## Dialogflow integration working on a Chatwoot instance using docker

https://user-images.githubusercontent.com/3526167/134868607-1cdf2c64-7d8b-41af-9db8-7157af1e5b4b.mov

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -33,6 +33,10 @@ WORKDIR /app
 
 COPY Gemfile Gemfile.lock ./
 
+# natively compile grpc and protobuf to support alpine musl (dialogflow-docker workflow)
```
Comment (sojan-official) [inline docker/Dockerfile:36]: let probably add the link to the github issue we raised in a comment as well :D

Comment (vishnu-narayanan) [reply docker/Dockerfile:36]: sure

**Comment by vishnu-narayanan** [comment]
Dialogflow integration working on a Chatwoot instance using docker

https://user-images.githubusercontent.com/3526167/134868607-1cdf2c64-7d8b-41af-9db8-7157af1e5b4b.mov

**Review by sojan-official — APPROVED** [review]

**Comment by vishnu-narayanan** [comment]
@sojan-official @pranavrajs Should we release a patch release for this or should we have this in the upcoming `v1.21.0 ` release ?

**Pull request merged**